requirements/edx/custom.txt

```diff
@@ -4,8 +4,8 @@
 -e git+https://github.com/edx-solutions/xblock-mentoring.git@8837eb5d91fed05ec4758dfd9b9e7adc5c906210#egg=xblock-mentoring
 -e git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.1.4#egg=xblock-image-explorer==1.1.4
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop.git@92ee2055a16899090a073e1df81e35d5293ad767#egg=xblock-drag-and-drop
-# FIXME: temp hash, until a proper solution is findout (ref ticket: https://edx-wiki.atlassian.net/browse/MCKIN-8249)
--e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@28c82e3d63f011029343115cb1b07e27afaeeb3e#egg=xblock-drag-and-drop-v2==2.2.2
+# FIXME: temp hash, until (https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/207 is merged into `master`)
```

Comment:
The situation is a lot more complex now, since there is greater divergence between the branches.

Reply:
Should we create a new branch from `28c82e3d63f011029343115cb1b07e27afaeeb3e` and cherry-pick this change there?

Reply:
There is a task to reconcile the branches, it just needs to be prioritised.